test_conformance/images/common.cpp

```diff
@@ -158,3 +158,10 @@ int get_format_list(cl_context context, cl_mem_object_type imageType,
     test_error(error, "Unable to get list of supported image formats");
     return 0;
 }
+
+size_t random_in_ranges(size_t minimum, size_t rangeA, size_t rangeB, MTdata d)
+{
+    if (rangeB < rangeA) rangeA = rangeB;
+    if (rangeA < minimum) return rangeA;
```

Comment:
Not very random :). This suggests the function is used in weird ways. Not worth thinking about in this PR.